recipes/salad/Fatoush.md

```diff
@@ -1,5 +1,17 @@
 # Fatoush Salad
 
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-70%25-orange.svg)
```

Comment:
it's mostly because of the dressing for the salad. it's pretty hard to get the right amount